gui/qt/main_window.py

```diff
@@ -1291,6 +1296,16 @@ def output_for_opreturn_stringdata(op_return):
         amount = 0
         return (TYPE_SCRIPT, ScriptOutput.from_string(script), amount)
 
+    @staticmethod
+    def output_for_opreturn_rawhex(op_return):
+        if not isinstance(op_return, str):
+            raise OPReturnError('OP_RETURN parameter needs to be of type str!')
+        op_return_script = b'\x6a' + bytes.fromhex(op_return)
```

Comment:
This ultimately comes from user input... which may not always be proper hex.

Decode failures will lead to an exception window (ValueError).

Please catch this exception and produce an intelligible error message for the user.  Exception windows are for when things go horribly wrong, not for normal UI activity, as I'm sure you'd agree. :)

Reply:
ah yes :-)

Reply:
Also suggest first stripping the string as obviously whitespace doesn't count if they checked "is hex".

`op_return = op_return.strip()` in this functions.

Reply:
And a tooltip on the checkbox itself would be nice. :)

Reply:
OK, should be fixed